pac4j-core/src/main/java/org/pac4j/core/authorization/authorizer/CheckHttpMethodAuthorizer.java

```diff
@@ -35,4 +35,16 @@ protected boolean check(final WebContext context, final CommonProfile profile, f
         final String requestMethod = context.getRequestMethod();
         return requestMethod.equalsIgnoreCase(element.toString());
     }
+
+    public static CheckHttpMethodAuthorizer requireAllPermissions(HTTP_METHOD... methods) {
```

Comment:
I think there is a failed copy/paste here. `checkHttpMethod`  is the right method name...

Reply:
Oops, sorry 😞 
Fixed